ci/gen_dox.sh

```diff
@@ -35,6 +35,12 @@
 #List of excluded driver folders for the documentation generation.
 EXCLUDE_DRV="talise navassa ad9083_api ad5940 madura"
 
+#List of folders that are handled differently due to their uncommon structure
+uncommon_drv_list=("sd-card" "imu" "api" "tdm")
```

Comment:
tdm should go under api, i can do that

Reply:
I made #2073, you can remote tdm from the list

Reply:
Yep, the tdm folder didn't make sense. Thanks. Will update the PR also.